src/components/CheckBoxTree/index.tsx

```diff
@@ -220,6 +232,10 @@ class CheckBoxTree extends React.Component<CheckBoxTreeProps> {
                 </Row>
                 <TreeNode headerContent={this.renderCheckAllButton()} />
                 {treeData.map((nodeData) => {
+                    const childrenHaveDifferentColors =
+                        !nodeData.children.every(
+                            (el) => el.color === nodeData.children[0].color
+                        );
```

Comment:
Bug: This check fails if children have differently-capitalized hex strings.

Repro:
http://localhost:9001/viewer?trajFileName=pc4covid19.simularium
1. Open the simularium file.
2. Open any object with children.
3. Paste `A384EE` into the first child's color picker. The parent should show the empty box.
4. Paste `a384EE` into the second child's color picker. The parent should still show the empty box.
5. Paste `A384EE` into the second child's color picker. Now, the parent will update with the correct color.